http/http/src/main/java/io/helidon/http/HeaderNameEnum.java

```diff
@@ -151,4 +151,84 @@ public String defaultCase() {
     public int index() {
         return index;
     }
+
+    static class Strings {
```

Comment:
What's the reasoning for this class ? Could HeaderNames be used instead ?

Reply:
Same question here. I also see in `HeaderNames`:

```
public static final String ACCEPT_NAME = Strings.ACCEPT_NAME;
```